### PR TITLE
fix(tree-item): preserve the alignment of tree-items with selection-mode: none regardless of whether they have children

### PR DESCRIPTION
**Pull request by Elijbet**
**Related Issue:** #5186

## Summary

Preserve the alignment of the `tree-items` regardless of whether they have children: outdent the disclosure arrows.

### CODE REVIEW

File: src/components/tree-item/tree-item.scss [hunk 1]
```diff
@@ -155,6 +149,19 @@
   }
 }
 
+// selection-mode none, dropdown without children
```
Comment (jcfranco) [inline src/components/tree-item/tree-item.scss:152]: IMO, the selector is readable, so we can drop this comment. Also, there's no dropdown here, right?

File: src/components/tree/tree.stories.ts [hunk 2]
```diff
@@ -61,6 +61,22 @@ export const simple = (): string => html`
   </calcite-tree>
 `;
 
+export const selectionModeNoneScales = stepStory(
```
Comment (jcfranco) [inline src/components/tree/tree.stories.ts:64]: Per our convo, can you replace this with a single test-only story that doesn't set scale (default)?

**Review by jcfranco — APPROVED** [review]
Aside from a few comments, this LGTM! 🎉

**Comment by jcfranco** [comment]
Adding @macandcheese for another pair of 👀 on this.

**Review by macandcheese — APPROVED** [review]
Looks good! - cc @ashetland for any necessary Figma changes.

**Pull request merged**